recipes/mds.rb

```diff
@@ -22,6 +22,26 @@
 
 cluster = 'ceph'
 
+if node['ceph']['version'] >= 'giant'
```

Comment:
The mds pools are not created by default for versions >= giant (before that they were automatically created, so there was no code to create the pools if not present).

After thinking about it, creating the pool "not_if" already exists should be safe and future proof. Want me to clean this up?